### PR TITLE
Add missing link in README

### PR DESCRIPTION
**Pull request by camelid**

### CODE REVIEW

**Comment by jyn514** [comment]
Looks like bors was confused.

@bors r=jonas-schievink rollup

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The Rust build system uses a Python script called `x.py` to build the compiler,
 which manages the bootstrapping process. More information about it can be found
 by running `./x.py --help` or reading the [rustc dev guide][rustcguidebuild].
 
+[rustc-dev-guide]: https://rustc-dev-guide.rust-lang.org
```
Comment (jyn514) [inline README.md:29]: Wait, what is this doing? There's no `rustc-dev-guide` link anywhere.

**Comment by camelid** [comment]
Hmm, the PR is still not in bors's queue.

File: README.md (hunk 1 above)
Comment (jyn514) [reply README.md:29]: Wow, that's a long way down 😆 can you move this link down there so it's clear what it's for?

**Comment by jyn514** [comment]
cc @rust-lang/infra - @bors is ignoring pings

File: README.md (hunk 1 above)
Comment (camelid) [reply README.md:29]: Sure. I just put it there because that's where the other dev guide links were :)

Comment (camelid) [reply README.md]: Okay, I moved it!

**Comment by jyn514** [comment]
Let's try this again.

@bors r+ rollup

**Comment by camelid** [comment]
The PR isn't even showing up in the unapproved queue...

**Comment by kennytm** [comment]
@bors retry 

```
2020-09-03T08:44:49.4142178Z curl: (56) OpenSSL SSL_read: Connection was reset, errno 10054
2020-09-03T08:44:49.4149901Z tar: Unexpected EOF in archive
2020-09-03T08:44:49.4150326Z tar: Unexpected EOF in archive
```

**Comment by camelid** [comment]
That's cool that rustbot automatically adds milestones! Thanks @Mark-Simulacrum!

**Pull request merged**